build-tools-internal/src/main/groovy/elasticsearch.build-scan.gradle

```diff
@@ -133,7 +133,7 @@ develocity {
         }
       } else {
         tag 'LOCAL'
-        if (providers.systemProperty('idea.active').present) {
```

Comment:
when running with configuration cache enabled, Project#providers is not available from a build scan background action. we can fall back to normal System.getProperty check instead as gradle instruments System.getProperty calls automatically to take into account for configuration caching.